docarray/array/storage/base/seqlike.py

```diff
@@ -63,6 +72,12 @@ def __bool__(self):
         return len(self) > 0
 
     def extend(self, values: Iterable['Document'], **kwargs) -> None:
+
+        from docarray.helper import check_root_id
+
+        if getattr(self, '_is_subindex', False):
```

Comment:
I think this `getattr` is not necessary anymore, we should be able to just to `self._is_subindex()`, right?